x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_enrichment/processors/grok/grok_ai_suggestions.tsx

```diff
@@ -109,14 +111,24 @@ const RefreshButton = ({
 };
 
 function useAiEnabled() {
-  const { dependencies } = useKibana();
-  const { observabilityAIAssistant } = dependencies.start;
+  const { dependencies, core } = useKibana();
+  const { observabilityAIAssistant, licensing } = dependencies.start;
 
   const aiAssistantEnabled = observabilityAIAssistant?.service.isEnabled();
 
   const genAiConnectors = observabilityAIAssistant?.useGenAIConnectors();
 
-  return aiAssistantEnabled && (genAiConnectors?.connectors || []).length > 0;
+  const aiEnabled = aiAssistantEnabled && (genAiConnectors?.connectors || []).length > 0;
+
+  const currentLicense = useObservable(licensing.license$);
+
+  const couldBeEnabled =
+    currentLicense?.hasAtLeast('enterprise') && core.application.capabilities.actions?.save;
```

Comment:
Not sure whether `actions.save` is the right capability or not - it seems like that's what you need to add an AI connector?